src/main/java/uk/gov/companieshouse/psc/delta/mapper/PscMapper.java

```diff
@@ -0,0 +1,223 @@
+                        encodedId));
+
+        if (source.getPscStatementId() != null) {
+            String encodedStatementId = MapperUtils.encode(source.getPscStatementId());
+            links.setStatements(String
+                    .format("/company/%s/persons-with-significant-control-statements/%s",
+                            source.getCompanyNumber(),
+                            encodedStatementId)); ;
+        }
+        target.setLinks(Collections.singletonList(links));
+    }
+
+    /**
+     * Manually map kind.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapKind(@MappingTarget Data target, Psc source) {
+        switch (source.getKind()) {
+            case INDIVIDUAL:
+                target.setKind("individual-person-with-significant-control");
+                break;
+            case CORPORATE_ENTITY:
+                target.setKind("corporate-entity-person-with-significant-control");
+                break;
+            case LEGAL_PERSON:
+                target.setKind("legal-person-person-with-significant-control");
+                break;
+            case SUPER_SECURE:
+                target.setKind("super-secure-person-with-significant-control");
+                break;
+            case INDIVIDUAL_BENEFICIAL_OWNER:
+                target.setKind("individual-beneficial-owner");
+                break;
+            case CORPORATE_BENEFICIAL_OWNER:
+                target.setKind("corporate-entity-beneficial-owner");
+                break;
+            case LEGAL_PERSON_BENEFICIAL_OWNER:
+                target.setKind("legal-person-beneficial-owner");
+                break;
+            case SUPER_SECURE_BENEFICIAL_OWNER:
+                target.setKind("super-secure-beneficial-owner");
+                break;
+            default:
+        }
+    }
+
+    /**
+     * Manually map CeasedOn.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapCeasedOn(@MappingTarget Data target, Psc source) {
+        if (source.getCeasedOn() != null) {
+            target.setCeasedOn(MapperUtils.parseLocalDate(source.getCeasedOn()));
+        }
+    }
+
+    /** encode psc_id. */
+    @AfterMapping
+    default void mapEncodedPscId(@MappingTarget ExternalData target, Psc source) {
+        if (source.getPscId() != null) {
+            target.setPscId(MapperUtils.encode(source.getPscId()));
+        }
+    }
+
+    /**
+     * Manually map ServiceAddressSameAsRegisteredOffice.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapRegisteredOfficeEnum(@MappingTarget Data target, Psc source) {
```

Comment:
I think this should be called mapServiceAddressSameAsRegisteredOffice for clarity